command_definitions.py

```diff
@@ -0,0 +1,128 @@
+import glob
+import re
+
+from fabric.api import run, put
+
+from host_setup import DOCKER_COMPOSE_EXECUTABLE, mkdir, provide_docker_compose
+
+
+class CallList(object):
+    def __init__(self, command, predicate=lambda: True):
+        self.command_list = command
+        self.base_command = ['echo']
```

Comment:
Let's make `base_command` a class variable (or class constant) rather than an instance variable. As the assigned values for the subclasses only depend on the respective subclass, not the instance, it can be a class variable/constant in the subclasses, too.